Twig/GridExtension.php

```diff
@@ -162,4 +176,42 @@ public function renderGrid(GridView $gridView)
             );
         }
     }
+
+    /**
+     * Render (only html) grid view
+     *
+     * @param GridView $gridView
+     *
+     * @return mixed
+     */
+    public function renderHtmlGrid(GridView $gridView)
+    {
+        if (!$gridView->getGrid()->isAjax()) {
+            return $this->renderBlock(
+                'grid_html',
+                array(
+                    'view' => $gridView
+                )
+            );
+        }
+    }
+
+    /**
+     * Render (only js) grid view
+     *
+     * @param GridView $gridView
+     *
+     * @return mixed
```

Comment:
Hi Pedro.
I Don't think so (or I'm missing some).
This method returns null or what the inner "renderBlock" calls returns. And as per your annotations, the "renderBlock" method returns "mixed".
Am I right?
Thanks...